miden-tx/src/kernel_tests/test_account.rs

```diff
@@ -53,19 +53,19 @@ pub fn test_set_code_is_not_immediate() {
         end
         ";
 
-    let transaction = prepare_transaction(tx_inputs, tx_args, code, None);
```

Comment:
I changed all functions that takes an `Option<File>` to instead receive only the code. This feature was seldomly used, and it makes more sense to have the test preparing the code instead of duplicating this code in multiple places (like it was prior)